crypto/evp/evp_fetch.c

```diff
@@ -155,8 +177,8 @@ static void destruct_method(void *method, void *data)
 }
 
 void *evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
-                        const char *name, const char *properties,
-                        void *(*new_method)(const char *name,
+                        int name_id, const char *properties,
```

Comment:
I've going back to expect a name here.  The reason is that a caller may mistakenly create a new number for an alias because the alias hasn't been given from any provider yet.